src/tests.rs

```diff
@@ -205,6 +291,7 @@ fn expressions_fmt() {
     info!(log, "message"; "f" => %f, "d" => ?d);
 }
 
+#[cfg(feature = "std")]
```

Comment:
Why is that needed?

Reply:
There were unrelated test failures (that's why I created two separate commits)